src/redux/sagas/expenditures/cancelStreamingPayment.ts

```diff
@@ -20,25 +25,48 @@ import {
 
 function* cancelStreamingPaymentAction({
   meta,
-  payload: { colonyAddress, streamingPayment, annotationMessage },
+  payload: { colonyAddress, streamingPayment, annotationMessage, shouldWaive },
 }: Action<ActionTypes.STREAMING_PAYMENT_CANCEL>) {
-  const batchKey = 'cancelStreamingPayment';
+  // To make the flow more readable
+  const isCancelMethod = !shouldWaive;
+  const isCancelAndWaiveMethod = shouldWaive;
+
+  const batchKey = `cancel${isCancelMethod ? '' : 'AndWaive'}StreamingPayment`;
```

Comment:
I think it's fine to use the same batch key, saves us an extra i18n message entry

Reply:
Oh, I thought we wanted to differentiate the ones that were just cancelled and ones that were cancelled + waived in places like the User Hub Transactions tab 🤔 @jakubcolony

Reply:
Now using the same batch key for both claimed & claimed+waived method calls @jakubcolony 

![Screenshot 2024-06-13 at 11 11 08](https://github.com/JoinColony/colonyCDapp/assets/50642296/9a2fa93b-96dc-44a4-8421-07afaedec5ac)
![Screenshot 2024-06-13 at 11 11 00](https://github.com/JoinColony/colonyCDapp/assets/50642296/79266de6-f674-4e4d-bd7a-f3b8b57d9b44)